README.md

```diff
@@ -1,41 +1,41 @@
-![Essential Next.js Build Plugin](next-on-netlify.png)
+![Next.js Runtime](next-on-netlify.png)
 
-# Essential Next.js Build Plugin
+# Next.js Runtime
 
 <p align="center">
-  <a aria-label="npm version" href="https://www.npmjs.com/package/@netlify/plugin-nextjs">
-    <img alt="" src="https://img.shields.io/npm/v/@netlify/plugin-nextjs">
+  <a aria-label="npm version" href="https://www.npmjs.com/package/@netlify/next-runtime">
+    <img alt="" src="https://img.shields.io/npm/v/@netlify/next-runtime">
   </a>
-  <a aria-label="MIT License" href="https://img.shields.io/npm/l/@netlify/plugin-nextjs">
-    <img alt="" src="https://img.shields.io/npm/l/@netlify/plugin-nextjs">
+  <a aria-label="MIT License" href="https://img.shields.io/npm/l/@netlify/next-runtime">
+    <img alt="" src="https://img.shields.io/npm/l/@netlify/next-runtime">
   </a>
 </p>
 
 ## What's new in this version
 
 Version 4 is a complete rewrite of the Essential Next.js plugin. For full details of everything that's new, check out
-[the v4 release notes](https://github.com/netlify/netlify-plugin-nextjs/blob/main/docs/release-notes/v4.md)
+[the v4 release notes](https://github.com/netlify/next-runtime/blob/main/docs/release-notes/v4.md)
 
-## Installing the plugin
+## Installing the Next.js Runtime
```

Comment:
[sand] We may want to consider moving this section further down the page. Again, as most folks rely on the automatic flow, this section will be noise to them. They're more likely to be interested in other content in the readme.